### PR TITLE
update fab installer to handle unspecified cloud settings

### PR DESCRIPTION
**Pull request by stugots**
Pylint:
* master today: 9.40 / 10 / 10
* this pull: 9.41 / 10 / 10

Changes:
* Provide a new second installation method. This is a partial installation (no tenant_admin, tenant, or cloud is created), but it requires no terminal interaction.
* Simplify some function calls in installer_fabfile.

I've tested this in my local dev environment, and on CentOS (both the full and partial installations).

### CODE REVIEW

**Comment by lexjacobs** [comment]
:+1:

File: INSTALL.md [hunk 1]
```diff
@@ -11,21 +11,59 @@ Before installing Goldstone, your server must meet the following prerequisites:
...
+
+```bash
+root# cd /opt/goldstone
+root# . bin/activate
+root# fab full_install
+```
+
+### 2. Partial (requires no terminal interaction)
+
+**N.B.: This section is a work in progress. It is not yet complete.**
```
Comment (jxstanford) [inline INSTALL.md]: I think nota bene falls under this category:  https://wiki.openstack.org/wiki/Documentation/Conventions#Avoid_obscure_non-English_words_and_abbreviations

The sentences are redundant.

File: installer_fabfile.py [hunk 2]
```diff
@@ -693,16 +680,70 @@ def install(pg_passwd='goldstone',  # pylint: disable=R0913
...
+            pg_passwd='goldstone',
+            django_admin_user='admin',
+            django_admin_email='root@localhost'):
+    """Do a partial installation of Goldstone.
+
+    This requires no interaction with the user.
+
+    This does what full_install() does, except for creating the Goldstone
+    tenant and OpenStack cloud rows in the Tenant and Cloud tables. After this
+    runs, the user must log in to Goldstone as a Django admin, create a
```
Comment (jxstanford) [inline installer_fabfile.py]: nits:
* refs to rows and tables are probably too much "sausage making." 
* as "the" Django admin?

**Comment by jxstanford** [comment]
I just had a couple doc comments, then :+1:

File: INSTALL.md (hunk 1 above)
Comment (stugots) [inline INSTALL.md]: Fixed.

File: installer_fabfile.py (hunk 2 above)
Comment (stugots) [inline installer_fabfile.py]: The -- Changed to "the."

Rows and tables -- I can't think of a better way to say this. Can you suggest something?

File: installer_fabfile.py [hunk 3]
```diff
@@ -693,16 +680,70 @@ def install(pg_passwd='goldstone',  # pylint: disable=R0913
...
+def install(django_admin_password,
+            pg_passwd='goldstone',
+            django_admin_user='admin',
+            django_admin_email='root@localhost'):
+    """Do a partial installation of Goldstone.
+
+    This requires no interaction with the user.
+
+    This does what full_install() does, except for creating the Goldstone
+    tenant and OpenStack cloud rows in the Tenant and Cloud tables. After this
```
Comment (jxstanford) [inline installer_fabfile.py]: This does what full_install() does, except for creating the Goldstone tenant and storing the OpenStack connection details.

**Pull request merged**